packages/react-table/src/demos/table-demos/ExpandCollapseAll.jsx

```diff
@@ -115,24 +126,53 @@ class ExpandCollapseAllTableDemo extends React.Component {
     });
   }
 
+  renderPagination(variant, isCompact) {
+    return (
+      <Pagination
+        isCompact={isCompact}
+        itemCount={36}
+        page={1}
+        perPage={10}
+        variant={variant}
+        titles={{
+          paginationTitle: `${variant} pagination`
+        }}
+      />
+    );
+  }
+
   render() {
     const { columns, rows, collapseAllAriaLabel } = this.state;
 
+    const tableToolbar = (
+      <Toolbar id="compact-toolbar">
+        <ToolbarContent>
+          <ToolbarItem variant="pagination">{this.renderPagination('top', true)}</ToolbarItem>
+        </ToolbarContent>
+      </Toolbar>
+    );
```

Comment:
I think I'd remove the toolbar if it only has pagination as the white space on the left seems odd. @mcarrano wdyt?

Reply:
We could also add some other controls to the toolbar?

Reply:
I agree with @mattnolting . But I'm OK to either remove it or add some simple filter actions. But did we say we would remove these toolbars until we worked out the correct structure to match core, or am I confusing this with another issue @tlabaj ?

Reply:
+1 @mattnolting @mcarrano, I think it would make sense to remove the toolbar here as well since we need to figure out the correct table structure and data first before implementing controls that would use the table data

Reply:
That was a different PR @mcarrano but it applies here as well.  We can add the toolbar back once we have a data set.